Client/Frontend/Home/HomepageMessageCardViewModel.swift

```diff
@@ -0,0 +1,111 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+protocol HomepageMessageCardProtocol {
```

Comment:
Just a thought - what this protocol provides is something all consumers of GleanPlumbMessages will need. If we can give it a more generalized name now, maybe we can reuse elsewhere when the time comes? We can keep it this way was well, and cross that bridge when we get there.

Reply:
Makes sense, I will update the name